documentation/4.0/content/samples/domains/model-in-image/_index.md

```diff
@@ -27,60 +36,52 @@ This sample demonstrates five Model in Image use cases:
 
 - [Initial]({{< relref "/samples/domains/model-in-image/initial.md" >}}): An initial WebLogic domain with the following characteristics:
 
-   - Image `model-in-image:WLS-v1` with:
-     - A WebLogic installation
+   - Auxiliary image `model-in-image:WLS-AI-v1` with:
```

Comment:
Since we are changing the default to use auxiliary images, I wonder if we should change the MII image name/tag to something like `model-in-image-ai:WLS-v1` or just `model-in-image:WLS-v1` and then have a different tag for the non-auxiliary image models.

Reply:
The sample directory structure, internal sample helper scripts, and testing solely rely on the ":" suffix throughout to distinguish images, so changing the predicate is impractical.

As for changing the suffix, how about we think about changing the current "WLS-v1" to "WLS-LI-v1", where LI indicates 'Layered Image' as opposed to 'Auxiliary Image'? We could then leave "WLS-AI-v1" unchanged. Plus this is more clear IMO, as then all tags would rigorously have a second component of either 'LI' or 'AI' that is meaningful rather than relying on knowing that a shorter tag indicates something important...

Note that this is a change that's not directly in the scope of this merge as the nomenclature was chosen well before 4.0 (plus the merge already occurred).

Reply:
Thoughts? @rjeberhard ?